tests/integration/kubernetes/confidential/confidential_common.sh

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env bats
+# Copyright 2022-2023 Advanced Micro Devices, Inc.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+source "../../tests_common.bash"
+
+export SUPPORTED_HYPERVISORS=(
+    "qemu-sev"
+    "qemu-snp"
+)
```

Comment:
I don't think this is going to work as you expect. :-)
This is an array, and I think just declaring it here, and then importing this file in the bats would be enough.

Reply:
Thanks, I have removed the export and I am loading the `confidential_common.sh` in the bats. And `confidential_common.sh` loads `tests_common.sh`